chat/src/main/java/org/navgurukul/chat/core/repo/AuthenticationRepository.kt

```diff
@@ -27,7 +27,7 @@ class AuthenticationRepository(
     private val appContext: Context
 ) {
 
-    suspend fun getLoginFlow(): LoginFlowResult? = suspendCoroutine {
+    private suspend fun getLoginFlow(): LoginFlowResult? = suspendCoroutine {
```

Comment:
Do not remove fake login part

Reply:
okay

Reply:
Didn't remove it just make this function as private please recheck.